app/models/pipeline_run_stage.rb

```diff
@@ -171,12 +188,11 @@ def prepare_dag(dag_name, attribute_dict, key_s3_params = nil)
                            attribute_dict,
                            pipeline_run.parse_dag_vars)
     self.dag_json = dag.render
-    copy_done_file = "echo done | aws s3 cp - #{sample.sample_output_s3_path}/\\$AWS_BATCH_JOB_ID.#{JOB_SUCCEEDED_FILE_SUFFIX}"
-    upload_dag_json_and_return_job_command(dag_json, dag_s3, dag_name, key_s3_params, copy_done_file)
+    save
+    upload_dag_json(dag_json, dag_s3)
   end
 
-  def host_filtering_command
```

Comment:
These `[stage name]_command` are called when the pipeline stage is kicked off (https://github.com/chanzuckerberg/idseq-web/blob/master/app/models/pipeline_run_stage.rb#L69), creating both the attributes for the dag json and job command. I split each of these `[stage name]_command` into two methods: `[stage name]_dag_attributes` and `[stage_name]_command`. The `_dag_attributes` methods creates the fields needed to create the dag, and `_command` methods creates the fields needed to run the dag.